cfgov/jinja2/v1/_includes/organisms/secondary-navigation.html

```diff
@@ -21,7 +21,7 @@
     <ul class="o-secondary-navigation_list
                o-secondary-navigation_list__parents">
         {%- for item in items %}
-            <li>
+            <li data-nav-is-active="{{ is_nav_active(request.url, item) }}">
```

Comment:
I don't think `is_nav_active` is necessary. You can do `{{ request.url == item.url }}`.